### PR TITLE
Add cloud note to schema package

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#181

- Adds note about not being able to use the `schema` package with cloud from the REPL
- Adds new `prepend` and `append` frontmatter.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -155,14 +160,41 @@ canonical: /path/to/canonical/doc/
...
+### v2 equivalent documentation
 To display a notice on a 1.x page that links to an equivalent 2.0 page,
 add the following frontmatter to the 1.x page:
 
 ```yaml
 v2: /influxdb/v2.0/get-started/
 ```
 
+### Prepend and append content to a page
+Use the `prepend` and `append` frontmatter to pre/append content to a page.
```
Comment (pierwill) [inline CONTRIBUTING.md:172]:
```suggestion
Use the `prepend` and `append` frontmatter to add content to the top of bottom of a page.
```

**Review by pierwill — APPROVED** [review]
Great, @sanderson! 👍 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (pierwill) [reply CONTRIBUTING.md:172]: Sorry should be `top or bottom of`...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -155,14 +160,41 @@ canonical: /path/to/canonical/doc/
...
 ```yaml
 v2: /influxdb/v2.0/get-started/
 ```
 
+### Prepend and append content to a page
+Use the `prepend` and `append` frontmatter to add content to the top or bottom of a page.
+Each has the following fields:
+
+- **block:** _(Optional)_ block style to wrap content in (note, warn, cloud, or enterprise)
+- **content:** _**(Required)**_ markdown content to pre/append.
```
Comment (kelseiv) [inline CONTRIBUTING.md:176]:
```suggestion
- **content:** _**(Required)**_ markdown content to add.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -155,14 +160,41 @@ canonical: /path/to/canonical/doc/
...
+
+```yaml
+append:
+  block: note
+  content: |
+    #### This is example markdown content
+    This is just an example note block that gets appended to the article.
+```
+
+Use this frontmatter with [cascade](#cascade) to pre/append the same content to
```
Comment (kelseiv) [inline CONTRIBUTING.md:186]:
```suggestion
Use this frontmatter with [cascade](#cascade) to add the same content to
```

File: content/influxdb/cloud/reference/flux/stdlib/influxdb-schema/_index.md [hunk 4]
```diff
@@ -13,6 +13,16 @@ menu:
...
+cascade:
+  append:
+    block: cloud
+    content: |
+      #### Supported in the InfluxDB Cloud UI
+      The `schema` package can retrieve schema information from InfluxDB Cloud when
+      used in the InfluxDB Cloud user interface (UI), but it **cannot** when using
+      the [Flux REPL](/influxdb/cloud/tools/repl/).
+      In the REPL, there is no way to provide the host, organization, and token
+      required to authorize with InfluxDB Cloud.
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/flux/stdlib/influxdb-schema/_index.md:25]:
```suggestion
           The `schema` package can retrieve schema information from the InfluxDB Cloud user interface (UI), but **not** from the [Flux REPL](/influxdb/cloud/tools/repl/).
```

**Review by kelseiv — APPROVED** [review]

**Pull request merged**